### PR TITLE
Update installation instructions for clang-format on Ubuntu 12.04.

### PR DESCRIPTION
**Pull request by tammoippen**

### CODE REVIEW

File: coding_guidelines_c++.md [hunk 1]
```diff
@@ -41,17 +41,29 @@ The code has to compile without warnings (in the default settings of the build i
 
 The [clang-format](http://clang.llvm.org/docs/ClangFormat.html) tool is build on the clang compile fronted. It allows to pretty print input files in a configurable manner, and has vim and emacs integration. We supply a [.clang-format](#clang-format-file) configuration file to enforce some parts of the coding style. During the code review process we check that there is no difference between the committed files and the formatted version of the committed files:
 
+Developer can benefit from the tool by formatting their changes before issuing a pull request: for fixing the formatting of a single file consider using `clang-format -i <committed file>` on that file. For fixing more files at once we provide a script that applies the formatting. From the source directory call:
```
Comment (heplesser) [inline coding_guidelines_c++.md]: "Developers"

**Comment by heplesser** [comment]
:+1: once the typo is fixed.

**Comment by jougs** [comment]
I agree with @heplesser. :+1: after the typo is fixed.

**Comment by tammoippen** [comment]
Typo fixed. Merging.

**Pull request merged**